build/BuildToolsTasks/BuildContentPipelineTask.cs

```diff
@@ -7,6 +7,17 @@ public sealed class BuildContentPipelineTask : FrostingTask<BuildContext>
 {
     public override void Run(BuildContext context)
     {
+        var buildPremake = new BuildPremake();
+        buildPremake.Run(context, "Content Pipeline", "native/pipeline", "pipeline.sln");
```

Comment:
Native content pipeline is being built by the MonoGame.Framework.Content.Pipeline.csproj, this is not needed.